bumble/device.py

```diff
@@ -1314,6 +1299,24 @@ async def power_off(self) -> None:
             await self.host.flush()
             self.powered_on = False
 
+    async def refresh_resolving_list(self) -> None:
+        assert self.keystore
+
+        await self.send_command(HCI_LE_Clear_Resolving_List_Command())  # type: ignore[call-arg]
```

Comment:
this command should only be sent if address resolution offloading is enabled.

Reply:
Done